docs/03_command-reference/eosio-cpp.md

```diff
@@ -71,4 +71,9 @@ compiler options:
   -sysroot=<string>        - Set the system root directory
   -v                       - Show commands to run and use verbose output
   -w                       - Suppress all warnings
+  -no-missing-ricardian-clause - Defaults to false, disables warnings for missing Ricardian clauses
 ```
+
+## Notes
+
+* -no-missing-ricardian-clause: Defaults to false, if enabled, it suppresses warnings for missing Ricardian clauses on contracts and contract actions. That includes the warnings generated when there is also no independent Ricardian clause file.
```

Comment:
Can enclose -no-missing-ricardian-clause in code font (backticks) as it's a command option:
-no-missing-ricardian-clause: Defaults to false, if enabled, it suppresses warnings for missing Ricardian clauses on contracts and contract actions. That includes the warnings generated when there is also no independent Ricardian clause file.

Reply:
Also, consider simplifying the last sentence to:

It also includes the warnings generated where there is no independent Ricardian clause file.